src/main/java/com/github/hcsp/Main.java

```diff
@@ -1,15 +1,15 @@
 package com.github.hcsp;
 
 public class Main {
-    public static String message = "Hello";
+    public static String message = "Hi";
 
     public static void main(String[] args) {
         System.out.println(message);
-        updateMessage();
+        updatemessage();
         System.out.println(message);
     }
 
-    public static void updateMessage() {
+    private static void updatemessage() {
```

Comment:
```suggestion
    public static void updatemessage() {
```

这里请不要修改，因为测试需要访问这个方法。